app/services/generate_claims_csv.rb

```diff
@@ -0,0 +1,19 @@
+require "csv"
+
+class GenerateClaimsCsv
+  include ServicePattern
+
+  HEADERS = %w[school_name school_urn amount claim_reference_number].freeze
+
+  def call
+    CSV.generate(headers: true) do |csv|
+      csv << HEADERS
+
+      Claim.find_each do |claim|
+        claim_amount = Claim::CalculateAmount.call(claim:)
```

Comment:
I realise @Nitemaeric [asked you](https://github.com/DFE-Digital/itt-mentor-services/pull/333#discussion_r1511406640) to move `claim_amount` into a Service.

My personal instinct would have been to implement it as a method on the `Claim` model directly, rather than as a standalone service. It feels like that would be a more 'obvious' way to get the claim amount.

For example, so I could call `claim.amount` and receive a `Money` object. And then call `.format` within the `GenerateClaimsCsv` service since that's a formatting/output concern. Something like:

```suggestion
        claim_amount = claim.amount.format(symbol: false, decimal_mark: ".", no_cents: false)
```

But this is just my opinion – no need to make any changes if you're happy with the current approach! 🙂